tests/testthat/test-score_forecasts.R

```diff
@@ -483,8 +556,8 @@ test_that("wis is correct, 2 intervals and median", {
     stringsAsFactors = FALSE
   )
 
-  actual <- score_forecasts(forecasts = test_forecasts, truth = test_truth)
-
+  actual <- score_forecasts(forecasts = test_forecasts, truth = test_truth, 
+                            use_median_as_point = TRUE)
```

Comment:
I suggest adding a test for WIS where use_median_as_point = FALSE, to ensure against a future recoding of score_forecasts() where the abs_error column is used to compute the WIS and it is computed from the point estimate.